server/collab/matchers/hist_matcher.py

```diff
@@ -14,43 +13,30 @@
 class HistogramMatcher(matcher.Matcher):
   @staticmethod
   def match(source, target):
-    start = time.time()
```

Comment:
They're not too useful. they were only to pinpoint the slow parts (which are mostly outside of that function, btw).